block_test.go

```diff
@@ -193,8 +199,9 @@ func genSeries(totalSeries, labelCount int, mint, maxt int64) []Series {
 	series := make([]Series, totalSeries)
 	for i := 0; i < totalSeries; i++ {
 		lbls := make(map[string]string, labelCount)
-		for len(lbls) < labelCount {
-			lbls[randString()] = randString()
+		lbls[defaultLabelName] = strconv.Itoa(i)
+		for j := 1; len(lbls) < labelCount; j++ {
```

Comment:
@krasi-georgiev I have lost touch with the codebase lately, does changing this functionality affect other places where it is used? Here the series are no longer random for every `genSeries` call, and it inclined more towards testing this particular PR.

Reply:
It is fine so far we don't need randomness just different label=value variations.
```
lbls[defaultLabelName+strconv.Itoa(j)] = defaultLabelValue + strconv.Itoa(j)
```